1_wqp_pull/cfg/wqp_partition_config.yml

```diff
@@ -3,5 +3,5 @@
 target_inv_size: 1000
 
 # Approximate maximum number of records to pull in each call to WQP.
-# Recommended number is 500000 unless that causes exceedance of max URL size allowed by the WAF
+# Recommended number is 25000 unless that causes exceedance of max URL size allowed by the WAF
```

Comment:
Wow, that's a lot smaller! Is this still required even after the improvements to WQP this year?

Reply:
Yeah this was originally just an oversight on my part (copied over from NWIS) but I don't think we _actually_ know what this number is.